distribution/docs/src/main/resources/_contents/_core-concepts/federation-intro-contents.adoc

```diff
@@ -0,0 +1,8 @@
+
+== Federation
+
+Federation is the ability of the ${branding} to query other data sources, including other ${branding}s.
+By default, the ${branding} is able to federate using OpenSearch, ${branding} 2.0 SOAP , ${branding} 1.3 SOAP , and ${branding} JSON protocols.
```

Comment:
we can remove all the other `protocols` here except OpenSearch. But we could add CSW